mk/config.mk

```diff
@@ -61,6 +61,12 @@ CFG_TEE_TA_LOG_LEVEL ?= 1
 # CFG_TEE_TA_LOG_LEVEL. Otherwise, they are not output at all
 CFG_TEE_CORE_TA_TRACE ?= y
 
+# When "y", the current thread will be shown
+CFG_TRACE_THREAD_ID ?= y
+
+# When "y", the function and line will be shown
+CFG_TRACE_FUNCTION ?= y
```

Comment:
Should one really want to remove such info from the traces? The `xMSG_RAW()` is used when func/line is useless.

Reply:
What do you mean Etienne? For the first question, yes I often want to remove that function and line. Are you saying that we can get rid of func:line in some other way when building or am I misunderstanding you here?

Reply:
`CFG_MSG_LONG_PREFIX_MASK = 0`

Reply:
Right, that was the one I was referring to in my first patch in this serie in the commit message that works but it's not very user friendly (I wrote CFG_MSG_LONG_PREFIX_THRESHOLD, which I can't even find in the code now, but it was that mask I had in mind). Even after reading [that comment](https://github.com/OP-TEE/optee_os/blob/master/mk/config.mk#L73-L78) a couple of times I find it hard to follow. I guess each bit represents long message (bit=1) for this bit field `|0:none|1:error|2:info|3:debug|4:flow|`? So the `0x1a (0b11010)` means, long on flow, debug and error? But sure, it does the job, so I can live with the bit field (to remember ...MASK = 0 is easy).